dotCMS/src/main/java/com/dotcms/rest/PushPublishResourceUtil.java

```diff
@@ -0,0 +1,44 @@
+package com.dotcms.rest;
+
+import com.dotcms.publisher.pusher.AuthCredentialPushPublishUtil;
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.util.Logger;
+
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+/**
+ * Provide util method for {@link IntegrityResource} and {@link BundleResource} classes
+ */
+public class PushPublishResourceUtil {
```

Comment:
Codacy found an issue: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)